__tests__/MoreScreen.test.tsx

```diff
@@ -64,18 +72,17 @@ beforeEach(() => {
 });
 
 describe('MorePage', () => {
-  it('toggles dark mode correctly', () => {
```

Comment:
You removed the test case completely why ?

Reply:
The more page was completely written so old test cases did not work anymore. I also needed to create a whole new test file.